playbooks/postgres_tuning.yml

```diff
@@ -1,13 +1,26 @@
 ---
 - name: PostgreSQL production tuning
-  hosts: ofn_servers
+  hosts:
```

Comment:
outside of the scope but I think this should be merged into the provisioning playbook soon. We'll need to parametrize it so that it works for all instances but I think it makes sense not to treat these tasks as an exception anymore.